vlib/v/fmt/tests/fn_headers_with_inline_comments_expected.vv

```diff
@@ -1,7 +1,8 @@
-fn main() { // main
+fn /* main */ main() {
```

Comment:
Oh God ...

Reply:
In my opinion, that should not be allowed, the `fn name_of_function` is very important for me, to find free functions everywhere, so for me, moving the comment after the function declaration is better, even though it will not preserve the original writer's intent.